src/controllers/controller.line.js

```diff
@@ -69,8 +69,8 @@ export default class LineController extends DatasetController {
 			const index = start + i;
 			const point = points[i];
 			const parsed = me.getParsed(index);
-			const x = xScale.getPixelForValue(parsed.x);
-			const y = reset ? yScale.getBasePixel() : yScale.getPixelForValue(_stacked ? me.applyStack(yScale, parsed) : parsed.y);
+			const x = xScale.getPixelForValue(parsed.x, i);
```

Comment:
This should be `index` instead of `i`?
Should the method be renamed to `getPixelForValueAndIndex` and make it private? (Not easily usable from external code, because you need to know the index)
Example use case that is not usable anymore: https://codepen.io/kurkle/pen/jOWBpmP

Reply:
Yes, I fixed the index. Also, I made `index` optional so that the use case you linked to should still work